common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -107,8 +134,19 @@ def get_transcripts_from_youtube(youtube_id, settings, i18n):
 
     youtube_text_api = copy.deepcopy(settings.YOUTUBE['TEXT_API'])
     youtube_text_api['params']['v'] = youtube_id
+    # if the transcript name is not empty on youtube server we have to pass
+    # name param in url in order to get transcript
+    # example http://video.google.com/timedtext?lang=en&v={VideoId}&name={transcript_name}
+    if youtube_transcript_name:
+        youtube_text_api['params']['name'] = youtube_transcript_name
     data = requests.get('http://' + youtube_text_api['url'], params=youtube_text_api['params'])
```

Comment:
if `youtube_transcript_name` is `''`, will this always fail the conditional in L145?